test/helpers/init.js

```diff
@@ -35,10 +35,11 @@ export async function getAtonomiTokenContract (owner, releaseAgent) {
 
 export async function getNetworkSettingsContract (app, owner) {
   const NetworkSettings = artifacts.require('NetworkSettings')
+  const Storage = await this.getStorageContract(owner);
   return app.createProxy(NetworkSettings, 'NetworkSettings', 'initialize', [
     owner,
     regFee, actFee,
-    repReward, repShare, blockThreshold]
+    repReward, repShare, blockThreshold, Storage.address]
```

Comment:
Dont use Storage.address.  Pass it in as a parameter to the function, and pass in ctx.contracts.storage.address instead.